src/handle_finalize.c

```diff
@@ -4,7 +4,6 @@ void handle_finalize(ethPluginFinalize_t *msg) {
     context_t *context = (context_t *) msg->pluginContext;
 
     msg->uiType = ETH_UI_TYPE_GENERIC;
-    msg->result = ETH_PLUGIN_RESULT_OK;
```

Comment:
this is merge mistake?

Reply:
We will check and reassign this value [here](https://github.com/LiskHQ/lisk-app-plugin/blob/8f3866c02bd9589ac6207220fa51a698e6d22b5b/src/handle_query_contract_ui.c#L93)